kumascript/src/render.js

```diff
@@ -210,6 +211,13 @@ async function render(
       if (!priorResult) {
         signatureToResult.set(token.signature, currentResult);
       }
+      if (process.env.BUILD_LOG_MACROS_USED) {
+        fs.appendFileSync(
+          process.env.BUILD_LOG_MACROS_USED,
+          `${macroName}|${token.args}\n`,
+          "utf-8"
+        );
+      }
```

Comment:
This won't work. It doesn't account for macros called only from within other macros. You want to move this code right after the following line in `kumascript/src/templates.js`:

https://github.com/mdn/kumascript/blob/affb2a7ca9f9789006111e97c71a8d57cf7f5762/src/templates.js#L94